tests/foreman/longrun/test_oscap.py

```diff
@@ -493,3 +495,129 @@ def test_positive_reporting_emails_of_oscap_reports():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.parametrize('distro', ['rhel8'])
+def test_positive_oscap_run_via_local_files(
+    module_org, default_proxy, content_view, lifecycle_env, distro, module_target_sat
+):
+    """End-to-End Oscap run via local files deployed with ansible
+
+    :id: 0dde5893-540c-4e03-a206-55fccdb2b9ca
+
+    :parametrized: yes
+
+    :customerscenario: true
+
+    :setup: scap content, scap policy , Remote execution
+
+    :steps:
+
+        1. Create a valid scap content
+        2. Import Ansible role theforeman.foreman_scap_client
+        3. Create a scap policy with ansible as deploy option
+        4. Associate the policy with a hostgroup
+        5. Run the Ansible job and then trigger the Oscap job.
+        6. Oscap must Utilize the local files for the client scan.
+
+    :expectedresults: Oscap run should happen using the --localfile argument.
+
+    :BZ: 2081777,2211952
+
+    :CaseImportance: Critical
+    """
+    SELECTED_ROLE = 'theforeman.foreman_scap_client'
+    file_name = 'security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2'
+    download_url = 'https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL8.xml.bz2'
```

Comment:
@adarshdubey-star Can you try changing the download_url as per our discussion? 
I've already updated that in 6.14.z and 6.13.z branch, and you can separately handle it in 6.12.z

Reply:
I have already raised a PR for 6.12 - https://github.com/SatelliteQE/robottelo/pull/12122/files but it looks like some more cherrypicks are missing to cause some failures, not related to scap plugin

Reply:
sure